PasswordReset/app/pwdmanager.py

```diff
@@ -58,7 +65,59 @@ def __kerberos_init():
         process.communicate()
         if process.returncode != 0:
             raise  KerberosInitFailed("Cannot retrieve kerberos ticket.")
-    
+######## Change Password using the url ###########
+#    def __change_password(self, uid, old_password, new_password, otp=None):
+#        """
+#        private function, use change_password instead
+#        """
+#        password_url = 'https://{0}/ipa/session/change_password'.format(self.current_host)
+#        headers = {
+#            'Content-Type': 'application/x-www-form-urlencoded',
+#            'Accept': 'text/plain',
+#        }
+#
+#        data = {
+#            'user': uid,
+#            'new_password': new_password,
+#            'old_password': old_password,
+#        }
+#        if otp:
+#            data['otp'] = otp
+#        response = self._session.post(
+#            password_url, headers=headers, data=data, verify=True
+#        )
+#
+#        if not response.ok:
+#            raise FreeIPAError(message=response.text, code=response.status_code)
+#
+#        pwchange_result = response.headers.get('X-IPA-Pwchange-Result', None)
+#        if pwchange_result != 'ok':
+#            if pwchange_result == 'invalid-password':
+#                raise SetPasswordFailed("Cannot update your password. You have entered {0}".format(pwchange_result))
+#            elif pwchange_result == 'policy-error':
+#                policy_error = response.headers.get('X-IPA-Pwchange-Policy-Error', None)
+#                raise SetPasswordFailed("Cannot update your password. {0}".format(policy_error))
+#            else:
+#                raise FreeIPAError(message=response.text, code=response.status_code)
+#        return response
+#
+#######################################################   
+
+############### Validate Password Policy #####################
+    def __vaidate_password(self, new_password):
+        policy = PasswordPolicy.from_names(
+            length=8,  # min length: 8
+            uppercase=2,  # need min. 2 uppercase letters
+            numbers=2,  # need min. 2 digits
+            special=1,  # need min. 2 special characters
```

Comment:
comment not in line with value :-)

Reply:
comment was from the initial developer and I just changed the value as per our need :D